geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/EnvironmentCommand.scala

```diff
@@ -24,11 +25,76 @@ class EnvironmentCommand(parent: JCommander) extends Command(parent) with LazyLo
 
   // TODO accumulo environment?
   override def execute(): Unit = {
-    if (params.sfts == null && params.converters == null) {
+    if (params.sfts == null && params.converters == null && !params.listSfts && !params.listConverters && !params.describeSfts && !params.describeConverters) {
       // default - list all
-      listSfts()
+      parent.usage(command)
+    } else if (params.listSfts){
+      listSftsNames()
+      if (params.listConverters){
+        println
```

Comment:
Seems like we could print this newline once above the if block and remove it from all of the nested blocks.